index.js

```diff
@@ -158,3 +160,83 @@ bot.onText(/\/meetups/, async msg => {
   );
   bot.sendMessage(msg.chat.id, out);
 });
+
+//Feature to save links and articles
+mongoose.connect("mongodb://localhost:27017/saved");
+const savedSchema = new Schema({
+  ChatID: Number,
+  name: String,
+  content: String
+});
+
+const Saved = mongoose.model("Saved", savedSchema);
+
+bot.on("message", async msg => {
+  //getting message string
+  let args = msg.text.split(" ");
+  const chatId = msg.chat.id;
+
+  //Save a note
+  if (args[0] == "/save") {
+    //Check if note with the same name already exists
+    await Saved.find({ ChatID: chatId, name: args[1] }, async (err, notes) => {
+      if (notes.length == 0) {
+        await Saved.create({
+          ChatID: chatId,
+          name: args[1],
+          content: args.splice(2, args.length).join(" ")
+        });
+        bot.sendMessage(chatId, "Note Saved");
+      } else {
+        bot.sendMessage(chatId, "Note already exists");
+      }
+    });
+  }
+  //View saved notes
+  if (args[0] == "/saved") {
+    await Saved.find({ ChatID: chatId }, (err, notes) => {
+      if (err) {
+        console.log(err);
+      }
+      if (notes.length == 0) {
+        bot.sendMessage(chatId, "No Saved Notes");
+      } else {
+        out = "Saved notes are : \n";
+        notes.map(note => {
+          out = out + "· /" + note.name + "\n";
+        });
+        bot.sendMessage(chatId, out);
+      }
+    });
+  }
+  //Delete saved notes
+  if (args[0] == "/delete") {
+    //Check if note exists
+    await Saved.find({ ChatID: chatId, name: args[1] }, async (err, notes) => {
+      if (notes.length == 0) {
+        bot.sendMessage(chatId, "Note doesn't exist");
+      } else {
+        await Saved.deleteOne({ ChatID: chatId, name: args[1] });
+        bot.sendMessage(chatId, "Note deleted");
+      }
+    });
+  }
+});
```

Comment:
🎉 🎉

Reply:
oops. Seems Github missed the lines I reviewed here. Actually, I meant to applaud for the Delete Note feature. Great work. 🎉